translations/en.json

```diff
@@ -1955,6 +1955,9 @@
                             "show_header_toggle": "Show Header Toggle?",
                             "toggle": "Toggle entities."
                         },
+                        "entity": {
```

Comment:
I am not sure I did Translations correctly. I can't seem to get them to show in the frontend. Just blanks..

Reply:
Change `src/translations/en.json`

Reply:
smh changed the wrong translations...